requirements/specifications/intents/user-interest.md

```diff
@@ -0,0 +1,299 @@
+# User Interest
```

Comment:
does this doc still belong in the intents folder?

Reply:
no, not really. I need to move it.

Reply:
Moved. kept the other doc around, but cleared it out. I'm writing it next week.